mmpose/datasets/pipelines/top_down_transform.py

```diff
@@ -253,14 +255,19 @@ def __init__(self,
                  sigma=2,
                  kernel=(11, 11),
                  valid_radius_factor=0.0546875,
-                 target_type='GaussianHeatMap',
+                 target_type='GaussianHeatmap',
                  encoding='MSRA',
                  unbiased_encoding=False):
         self.sigma = sigma
         self.unbiased_encoding = unbiased_encoding
         self.kernel = kernel
         self.valid_radius_factor = valid_radius_factor
-        self.target_type = target_type
+        if target_type == 'GaussianHeatMap':
```

Comment:
Ask for opinion: is it safe to compare the lower cased string?

Reply:
If so, we can avoid the deprecation warning

Reply:
The lower case `gaussianheatmap` looks strange.

Reply:
Yeah, how about we do not use it but we allow it?

Reply:
OK.